Resources/Prototypes/_Impstation/Shaders/shaders.yml

```diff
@@ -0,0 +1,4 @@
+- type: shader
+  id: AccesibleFullStealth
```

Comment:
im sorry to be that guy but you missed an 's' in accessible

Reply:
it's never been more over (will fix!)